core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1962,6 +1962,20 @@ private[spark] object Utils extends Logging {
     path
   }
 
+  /**
+   * Updates Spark config with properties from a set of Properties.
+   * Provided properties have the highest priority.
+   */
+  def updateSparkConfigFromProperties(conf: SparkConf, properties: Map[String, String])
+    : Unit = {
+    properties.filter { case (k, v) =>
+      k.startsWith("spark.")
```

Comment:
I just re-used the code found [here](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/util/Utils.scala#L1952).